robot_io/RegisterInfo info
bool[] bits
